lib/mongoid_session_store/store.rb

```diff
@@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
   private
 
     def generate_sid
-      BSON::ObjectId.new.to_s
+      SecureRandom.hex(32)
```

Comment:
Ah, sorry!

How do I do that?

On Fri, Jun 24, 2016 at 10:29 AM, Máximo Mussini <notifications@github.com>
wrote:

> In lib/mongoid_session_store/store.rb
> <https://github.com/SolarCS/mongo_session_store/pull/6#discussion_r68415604>
> :
>
> > @@ -9,7 +9,7 @@ class Store < ActionDispatch::Session::AbstractStore
> >    private
> >
> >      def generate_sid
> > -      BSON::ObjectId.new.to_s
> > +      SecureRandom.hex(32)
>
> You should unwatch the repo. This is a public repo 😃
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/SolarCS/mongo_session_store/pull/6/files/e03633243e28193609e7532b65d816ee43944ee6#r68415604>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIGVVBI9-hpl74GvwJ2TO1cnADWcRzv3ks5qO_fvgaJpZM4I9sS0>
> .
>



-- 

Matt Danilo

*Account Strategist*

mdanilo@cipherhealth.com | 646.568.6143

CipherHealth <http://www.cipherhealth.com/> | 555 8th Avenue, Suite 701,
New York, NY

*The information contained in this email message is intended only for the
use of the individual or the entity named above. If the reader of this
message is not the intended recipient, or the employee or agent responsible
for delivering it to the intended recipient, you are hereby notified that
any dissemination, distribution or copying of this communication is
strictly prohibited. If you have received this communication in error,
please immediately notify me by email (mdanilo@cipherhealth.com
<mdanilo@cipherhealth.com>), and destroy the original message. Thank you.*